test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -186,41 +270,45 @@ TEST_F(AmetsuchiTest, SampleTest) {
     }
   });
 
-  blocks->getAccountTransactions("admin1").subscribe([](auto tx) {
-    EXPECT_EQ(tx.creator_account_id, "admin1");
-    EXPECT_EQ(tx.commands.size(), 3);
-  });
-  blocks->getAccountTransactions("admin2").subscribe([](auto tx) {
-    EXPECT_EQ(tx.creator_account_id, "admin2");
-    EXPECT_EQ(tx.commands.size(), 4);
-  });
-
-  // request for non-existing user
-  auto number_of_calls = 0;
-  auto getAccountsTxWrapper = make_test_subscriber<CallExact>(
-      blocks->getAccountTransactions("non_existing_user"), 0);
-  getAccountsTxWrapper.subscribe(
-      [&number_of_calls](auto val) { number_of_calls++; });
-
-  ASSERT_TRUE(getAccountsTxWrapper.validate());
-  ASSERT_EQ(number_of_calls, 0);
-
-  blocks->getAccountAssetTransactions("user1@ru", "RUB#ru")
-      .subscribe([](auto tx) { EXPECT_EQ(tx.commands.size(), 1); });
-  blocks->getAccountAssetTransactions("user2@ru", "RUB#ru")
-      .subscribe([](auto tx) { EXPECT_EQ(tx.commands.size(), 1); });
+  {
+    auto wrapper = make_test_subscriber<EqualToList>(
```

Comment:
Better to rework tests as separated small pieces for improving readability and supporting of code.

Reply:
Yes, but this function exists before this PR. The reason for this change in this PR is expected `commands.size()` is different now because of some changes in the API specification.

Reply:
@muratovv ?

Reply:
Ok, good idea.